src/core/modes/walk-mode.ts

```diff
@@ -59,18 +59,37 @@ export class WalkMode implements Mode {
       return;
     }
 
-    const position = firstVisibleRange.start;
-    const range = new vscode.Range(position, position);
+    let visibleStartLine = firstVisibleRange.start;
+    let visibleEndLine = firstVisibleRange.start;
+    const junbaeLocation = { top: '5%', 'font-size': '50px' };
+    const location = vscode.workspace.getConfiguration('junbae-mode').get('location');
+
+    if (location === 'top') {
+      visibleStartLine = firstVisibleRange.start;
+      visibleEndLine = firstVisibleRange.start;
+    }
+    if (location === 'bottom') {
+      visibleStartLine = firstVisibleRange.end;
+      visibleEndLine = firstVisibleRange.end;
+      junbaeLocation.top = '-100px';
+      junbaeLocation['font-size'] = '100px';
+    }
+
+    const range = new vscode.Range(visibleStartLine, visibleEndLine);
 
     if (this.combo !== this.renderedComboCount || !range.isEqual(this.renderedRange!)) {
       this.renderedComboCount = this.combo;
       this.renderedRange = range;
-      this.createWalkMotionDecorator([range], editor);
+      this.createWalkMotionDecorator([range], editor, junbaeLocation);
       this.createTimerDecoration([range], editor);
     }
   }
 
-  private createWalkMotionDecorator = (ranges: vscode.Range[], editor: vscode.TextEditor) => {
+  private createWalkMotionDecorator = (
+    ranges: vscode.Range[],
+    editor: vscode.TextEditor,
+    junbaeLocation: { top: string },
```

Comment:
Why only `top`?
you give font-size too... 
If you thought about another options(includes `font-size`), you may need to set the parameters differently

Reply:
I see